pkg/geo/wkt/BUILD.bazel

```diff
@@ -22,3 +22,20 @@ go_test(
         "@com_github_twpayne_go_geom//:go-geom",
     ],
 )
+
+# Based on pkg/geo/wkt/generate.sh file
+genrule(
+    name = "wtk-generated",
+    srcs = [
+        "wkt.y",
+    ],
+    outs = ["wkt_generated.go"],
+    cmd = """
+    $(location @org_golang_x_tools//cmd/goyacc) -o $(location wkt_generated.go) -p "wkt" $(location wkt.y)
+    cat $(location wkt_generated.go) | sed -e 's/wktErrorVerbose = false/wktErrorVerbose = true/' > wkt_generated.go.tmp
```

Comment:
Use the `-i` option to `sed` so you don't have to make a temporary file yourself.

Reply:
`-i` works different on OSX and linux :\ (`-i ""` in OSX, `-i` in linux) -- theyre not compatible...

Reply:
^^
Yes I tried that and make it work, but at my local (OSX) adding the an additional flag.

So I think is better to stay with this rename file strategy to avoid issues with `-i` as @otan said

Reply:
i love technology 🙄